data/helpers.d/getopts

```diff
@@ -193,6 +202,8 @@ ynh_handle_getopts_args () {
 
 				# Escape double quote to prevent any interpretation during the eval
 				arguments[$i]="${arguments[$i]//\"/\\\"}"
+				# Escape $ as well to prevent the string following it to be seen as a variable.
+				all_args[$i]="${all_args[$i]//$/\\\$}"
```

Comment:
Shouldn't you escape `arguments` here...?

Reply:
Oups indeed, bad copy paste !